packages/types/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@comunica/types",
+  "version": "1.19.2",
+  "description": "Typings module for Comunica",
+  "lsd:module": false,
+  "main": "index.js",
+  "typings": "index",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/comunica/comunica.git",
+    "directory": "packages/types"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "keywords": [
+    "comunica",
+    "types"
+  ],
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/comunica/comunica/issues"
+  },
+  "homepage": "https://comunica.dev/",
+  "files": [
+    "components",
+    "lib/**/*.d.ts",
+    "lib/**/*.js",
+    "bin/**/*.d.ts",
+    "bin/**/*.js",
+    "index.d.ts",
+    "index.js"
+  ],
+  "devDependencies": {
```

Comment:
All of these would have to become actual dependencies based on how the types are defined now.

This makes me wonder if we really want this, or if we somehow want to define some more minimal interfaces of these packages internally, without having our types package blow up in number of interfaces.

Reply:
I share this concern, although I have no idea of which percentage of these modules' typings we're actually using. I'll try to have a go at this and report back here.

Reply:
Gave a go at this one but found that Comunica is using quite a lot of both `AsyncIterator` and Immutable's `Map`, which would require quite a bit of typings duplication. Any ideas?

Reply:
Yeah, I was afraid of that. Let's just keep it as-is then.